frontend/src/auth.js

```diff
@@ -158,6 +158,18 @@ async function api_category_download(id) {
   return res;
 }
 
+async function api_get_users() {
+  const res = await api_call("/user/users", {
```

Comment:
To keep this consistent, we should move the URL into `constants/links.js` and reference it here.